cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+timeout: 2700s
+steps:
+  # This step runs the python script to build and push the Docker image.
+  # We use the gcloud builder because it comes with python and the docker client.
+  - name: "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud"
+    entrypoint: "python3"
+    args:
+      - "./dev/tools/push-images"
+      # Assuming the script accepts project and tag arguments.
+      # Using standard Cloud Build substitutions.
+      - "--image-prefix=${_IMAGE_PREFIX}/"
+
+# The `images` section lists the container images built by this pipeline.
```

Comment:
Do we need to list them here?  I think `push-images` will _also_ try to push the images.

I'm not sure who is supposed to do the pushing: CloudBuild or the Job